content/docs/variables.json

```diff
@@ -1,3 +1,3 @@
 {
-    "cert_manager_latest_version": "v1.14.3"
+    "cert_manager_latest_version": "v1.14.4"
```

Comment:
Update the install versions in or set up variables like this, for those versions:
 * https://cert-manager.io/v1.12-docs/installation/
 * https://cert-manager.io/v1.13-docs/installation/

Reply:
I bumped those versions in https://github.com/cert-manager/website/pull/1441